sdk/storage/Azure.Storage.Common/src/Azure.Storage.Common.csproj

```diff
@@ -1,6 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <TargetFrameworks>$(RequiredTargetFrameworks);net6.0</TargetFrameworks>
```

Comment:
Do we need this in the specific client libraries also?
Can we add this to the storage test projects?

Reply:
I'm going to revert changes in `Azure.Storage` to minimize the size of the PR and send those as a separate one.